src/devices/Tlc1543/samples/README.md

```diff
@@ -0,0 +1,4 @@
+# Sample
+This sample shows
```

Comment:
Maybe you can extract few code elements, it's always nice to have some in the documentation